keps/sig-storage/2589-csi-migration-portworx/README.md

```diff
@@ -50,6 +56,33 @@ Major milestones for Portworx in-tree plugin CSI migration:
 
 - 1.23
   - Portworx CSI migration to Alpha
-- 1.24
+- 1.25
   - Portworx CSI migration to Beta, off by default
 
+## Design details
+
+### Test Plan
+
+[ ] I/we understand the owners of the involved components may require updates to
+existing tests to make this code solid enough prior to committing the changes necessary
+to implement this enhancement.
+
+##### Prerequisite testing updates
+
+No additional tests are needed, rather the issue is orchestrating CSI driver
+deployment for prow jobs. This has been complicated by the storage provider
+extraction work, which no longer permits storage provider specific orchestration
+in the k/k repository. This means that it is not possible to run any test for
+portworx-volume in k/k.
+
+##### Unit tests
+
+N/A
```

Comment:
Can you add this one: https://github.com/kubernetes/csi-translation-lib/blob/master/plugins/portworx_test.go